docs/roles/vice-president.rst

```diff
@@ -12,22 +12,27 @@ responsibilities and role of the vice president.
 Description
 -----------
 
-< general description of role, 1-2 paragraphs >
+The Vice President's primary role is to assist the president where possible. This generally includes helping write and send out emails through the github.io repo as well as helping to manage the presentation schedule. These tasks are generally split evenly between the President and Vice Presient. Making sure these two items continue smoothly is vital and is the thus is the primary responsiblity. 
+
+This role is largely picking up slack where the President needs additional help. Items can include anything from fundraising to event planning. Historically, the vice president has taken on external event planning, such as fund raising and expo presentations/displays.
 
```

Comment:
The [style guide](https://documentation-style-guide-sphinx.readthedocs.io/en/latest/index.html) for Sphinx reST docs is 79 characters per line. Could you reformat the line width in the doc to follow the style guide?